### PR TITLE
Use a regex to match internal packages

### LINKED ISSUE #65: Stack frames starting `com.google.android` appear as 'app' stack frames in UI
As you can see in the screenshot, stack-frames that begin with `com.google.android` are blue, and are therefore treated as application code in the Sentry UI.

google_callstack

We should filter out the stack-frames that start with `com.google.android`.

PR incoming.

### PR DESCRIPTION
**Pull request by marcomorain**
- Use a regex to match internal packages
- Add tests for the regex since it's hard to read a regex
- Add com.google.android to the list of internal packages
- Add filenames to callstack JSON packages

Fixes #65

### CODE REVIEW

**Comment by marcomorain** [comment]
tests++

File: sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java [hunk 1]
```diff
@@ -941,42 +950,42 @@ public static JSONObject getStackTrace(Throwable t) throws JSONException {
...
 
-                frame.put("in_app", inApp);
+			int lineno = ste.getLineNumber();
+			if (!ste.isNativeMethod() && lineno >= 0) {
+				frame.put("lineno", lineno);
+			}
 
-                frameList.put(frame);
-            }
+			String className = ste.getClassName();
```
Comment (marcomorain) [inline sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java]: Looks like Sentry.java has a mix of spaces and tabs in different parts of the file.

**Comment by marcomorain** [comment]
@joshdholtz any  time to review this change?

**Comment by marcomorain** [comment]
Hi @joshdholtz  - any interest in this change?

**Comment by marcomorain** [comment]
Ping @joshdholtz - any interest in these few PRs?

**Comment by joshdholtz** [comment]
OMG tests! I've been wanting to start adding some tests 😊

**Pull request merged**